### PR TITLE
[15.0][MIG] delivery_automatic_print

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

File: delivery_automatic_print/models/report_print.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def button_validate(self):
+        res = super(StockPicking, self).button_validate()
+        for pickings in self:
```
Comment (manengel) [inline delivery_automatic_print/models/report_print.py:9]: I am unsure if this is missing of that or not: I see no check that these documents have been created before. Question is: Do you even want such a check?

**Comment by manengel** [comment]
Why was that migrated at the same time as carrier_shipping_label_template? Anyways, I will check this module before continuing here.

**Comment by manengel** [comment]
@nikunjantala Would appreciate to receive feedback from you there.

**Comment by nikunjantala** [comment]
Both module developed same time. It is inter dependent. If base_report_to_printer is installed and user click on Validate button in Picking, then delivery_automatic_print will automatic send Delivery Slip and Shipping Label to Printer for print.

File: delivery_automatic_print/models/report_print.py (hunk 1 above)
Comment (jans23) [reply delivery_automatic_print/models/report_print.py:9]: @nikunjantala ?

Comment (manengel) [reply delivery_automatic_print/models/report_print.py:9]: @nikunjantala We would be glad if you could provide feedback.

Comment (nileshsheliya) [reply delivery_automatic_print/models/report_print.py:9]: @manengel Yes, We are checking that if that picking is delivery order, then system send Shipping label attached(UPS or Deutsche post) and Delivery order report to Printer for print.

Comment (nikunjantala) [reply delivery_automatic_print/models/report_print.py:9]: @manengel Yes, We are checking that if that picking is delivery order, then system send Shipping label attached(UPS or Deutsche post) and Delivery order report to Printer for print.

**Comment by alihashaam** [comment]
@jans23 @nikunjantala 
since we dont use labels here, is this PR ready for review?

**Comment by alihashaam** [comment]
@jans23 this PR can be merged now

**Comment by jans23** [comment]
Now the CI fails. @alihashaam can you check this?
https://github.com/Nitrokey/odoo-modules/actions/runs/4133120636

**Comment by alihashaam** [comment]
@jans23 sure, I am checking it

**Pull request merged**